CMakeLists.txt

```diff
@@ -19,9 +19,8 @@ else()
 endif()
 
 if (MSVC)
-  if (BUILD_SHARED_LIBS)
-    target_compile_definitions(lvgl PUBLIC LV_ATTRIBUTE_EXTERN_DATA=__declspec\(dllimport\))
-  else()
-    target_compile_definitions(lvgl PUBLIC LV_ATTRIBUTE_EXTERN_DATA=__declspec\(dllexport\))
-  endif()
-endif()
\ No newline at end of file
+  target_compile_definitions(lvgl
+    PRIVATE LV_ATTRIBUTE_EXTERN_DATA=__declspec\(dllimport\)
+    PUBLIC LV_ATTRIBUTE_EXTERN_DATA=__declspec\(dllexport\)
```

Comment:
Using both `PRIVATE` and `PUBLIC` is not that intuitive. Please add some comments to explain what a happens here and why we do this.

Reply:
@kisvegabor  I added some explanations. I hope this makes it clearer.